shared/fixes/bash/ensure_redhat_gpgkey_installed.sh

```diff
@@ -14,6 +14,8 @@ then
   # (to ensure there won't be e.g. CRC error).
   IFS=$'\n' GPG_OUT=($(gpg --with-fingerprint "${REDHAT_RELEASE_KEY}" | grep 'Key fingerprint ='))
   GPG_RESULT=$?
+  # Reset IFS back to default
+  unset $IFS
```

Comment:
Shouldn't this be just `unset IFS`?

Reply:
Interesting point.  It looks like I was working from an old code base, which upon further examination, was inconsistent in the use of the unset command:
```
# grep 'unset IFS' *
audit_rules_execution_chcon.sh:	unset IFS
audit_rules_execution_chcon.sh:	unset IFS
audit_rules_execution_chcon.sh:				unset IFS
audit_rules_execution_restorecon.sh:	unset IFS
audit_rules_execution_restorecon.sh:	unset IFS
audit_rules_execution_restorecon.sh:				unset IFS
audit_rules_execution_semanage.sh:	unset IFS
audit_rules_execution_semanage.sh:	unset IFS
audit_rules_execution_semanage.sh:				unset IFS
audit_rules_execution_setsebool.sh:	unset IFS
audit_rules_execution_setsebool.sh:	unset IFS
audit_rules_execution_setsebool.sh:				unset IFS
audit_rules_login_events_faillock.sh:	unset IFS
audit_rules_login_events_lastlog.sh:	unset IFS
audit_rules_login_events_tallylog.sh:	unset IFS
audit_rules_privileged_commands_chage.sh:	unset IFS
audit_rules_privileged_commands_chage.sh:	unset IFS
audit_rules_privileged_commands_chage.sh:				unset IFS
audit_rules_privileged_commands_chsh.sh:	unset IFS
audit_rules_privileged_commands_chsh.sh:	unset IFS
audit_rules_privileged_commands_chsh.sh:				unset IFS
audit_rules_privileged_commands_crontab.sh:	unset IFS
audit_rules_privileged_commands_crontab.sh:	unset IFS
audit_rules_privileged_commands_crontab.sh:				unset IFS
audit_rules_privileged_commands_gpasswd.sh:	unset IFS
audit_rules_privileged_commands_gpasswd.sh:	unset IFS
audit_rules_privileged_commands_gpasswd.sh:				unset IFS
audit_rules_privileged_commands_newgrp.sh:	unset IFS
audit_rules_privileged_commands_newgrp.sh:	unset IFS
audit_rules_privileged_commands_newgrp.sh:				unset IFS
audit_rules_privileged_commands_pam_timestamp_check.sh:	unset IFS
audit_rules_privileged_commands_pam_timestamp_check.sh:	unset IFS
audit_rules_privileged_commands_pam_timestamp_check.sh:				unset IFS
audit_rules_privileged_commands_passwd.sh:	unset IFS
audit_rules_privileged_commands_passwd.sh:	unset IFS
audit_rules_privileged_commands_passwd.sh:				unset IFS
audit_rules_privileged_commands_postdrop.sh:	unset IFS
audit_rules_privileged_commands_postdrop.sh:	unset IFS
audit_rules_privileged_commands_postdrop.sh:				unset IFS
audit_rules_privileged_commands_postqueue.sh:	unset IFS
audit_rules_privileged_commands_postqueue.sh:	unset IFS
audit_rules_privileged_commands_postqueue.sh:				unset IFS
audit_rules_privileged_commands_ssh_keysign.sh:	unset IFS
audit_rules_privileged_commands_ssh_keysign.sh:	unset IFS
audit_rules_privileged_commands_ssh_keysign.sh:				unset IFS
audit_rules_privileged_commands_sudoedit.sh:	unset IFS
audit_rules_privileged_commands_sudoedit.sh:	unset IFS
audit_rules_privileged_commands_sudoedit.sh:				unset IFS
audit_rules_privileged_commands_sudo.sh:	unset IFS
audit_rules_privileged_commands_sudo.sh:	unset IFS
audit_rules_privileged_commands_sudo.sh:				unset IFS
audit_rules_privileged_commands_su.sh:	unset IFS
audit_rules_privileged_commands_su.sh:	unset IFS
audit_rules_privileged_commands_su.sh:				unset IFS
audit_rules_privileged_commands_umount.sh:	unset IFS
audit_rules_privileged_commands_umount.sh:	unset IFS
audit_rules_privileged_commands_umount.sh:				unset IFS
audit_rules_privileged_commands_unix_chkpwd.sh:	unset IFS
audit_rules_privileged_commands_unix_chkpwd.sh:	unset IFS
audit_rules_privileged_commands_unix_chkpwd.sh:				unset IFS
audit_rules_privileged_commands_userhelper.sh:	unset IFS
audit_rules_privileged_commands_userhelper.sh:	unset IFS
audit_rules_privileged_commands_userhelper.sh:				unset IFS
# grep 'unset $IFS' *
audit_rules_dac_modification_chmod.sh:        unset $IFS
audit_rules_dac_modification_chmod.sh:        unset $IFS
audit_rules_dac_modification_chmod.sh:                                unset $IFS
audit_rules_dac_modification_chown.sh:        unset $IFS
audit_rules_dac_modification_chown.sh:        unset $IFS
audit_rules_dac_modification_chown.sh:                                unset $IFS
audit_rules_dac_modification_fchmodat.sh:        unset $IFS
audit_rules_dac_modification_fchmodat.sh:        unset $IFS
audit_rules_dac_modification_fchmodat.sh:                                unset $IFS
audit_rules_dac_modification_fchmod.sh:        unset $IFS
audit_rules_dac_modification_fchmod.sh:        unset $IFS
audit_rules_dac_modification_fchmod.sh:                                unset $IFS
audit_rules_dac_modification_fchownat.sh:        unset $IFS
audit_rules_dac_modification_fchownat.sh:        unset $IFS
audit_rules_dac_modification_fchownat.sh:                                unset $IFS
audit_rules_dac_modification_fchown.sh:        unset $IFS
audit_rules_dac_modification_fchown.sh:        unset $IFS
audit_rules_dac_modification_fchown.sh:                                unset $IFS
audit_rules_dac_modification_fremovexattr.sh:        unset $IFS
audit_rules_dac_modification_fremovexattr.sh:        unset $IFS
audit_rules_dac_modification_fremovexattr.sh:                                unset $IFS
audit_rules_dac_modification_fsetxattr.sh:        unset $IFS
audit_rules_dac_modification_fsetxattr.sh:        unset $IFS
audit_rules_dac_modification_fsetxattr.sh:                                unset $IFS
audit_rules_dac_modification_lchown.sh:        unset $IFS
audit_rules_dac_modification_lchown.sh:        unset $IFS
audit_rules_dac_modification_lchown.sh:                                unset $IFS
audit_rules_dac_modification_lremovexattr.sh:        unset $IFS
audit_rules_dac_modification_lremovexattr.sh:        unset $IFS
audit_rules_dac_modification_lremovexattr.sh:                                unset $IFS
audit_rules_dac_modification_lsetxattr.sh:        unset $IFS
audit_rules_dac_modification_lsetxattr.sh:        unset $IFS
audit_rules_dac_modification_lsetxattr.sh:                                unset $IFS
audit_rules_dac_modification_removexattr.sh:        unset $IFS
audit_rules_dac_modification_removexattr.sh:        unset $IFS
audit_rules_dac_modification_removexattr.sh:                                unset $IFS
audit_rules_dac_modification_setxattr.sh:        unset $IFS
audit_rules_dac_modification_setxattr.sh:        unset $IFS
audit_rules_dac_modification_setxattr.sh:                                unset $IFS
audit_rules_file_deletion_events.sh:        unset $IFS
audit_rules_file_deletion_events.sh:        unset $IFS
audit_rules_file_deletion_events.sh:                                unset $IFS
audit_rules_kernel_module_loading.sh:        unset $IFS
audit_rules_kernel_module_loading.sh:        unset $IFS
audit_rules_kernel_module_loading.sh:                                unset $IFS
audit_rules_kernel_module_loading.sh:        unset $IFS
audit_rules_kernel_module_loading.sh:        unset $IFS
audit_rules_kernel_module_loading.sh:        unset $IFS
audit_rules_login_events.sh:        unset $IFS
audit_rules_login_events.sh:        unset $IFS
audit_rules_login_events.sh:        unset $IFS
audit_rules_mac_modification.sh:        unset $IFS
audit_rules_media_export.sh:        unset $IFS
audit_rules_media_export.sh:        unset $IFS
audit_rules_media_export.sh:                                unset $IFS
audit_rules_networkconfig_modification.sh:        unset $IFS
audit_rules_networkconfig_modification.sh:        unset $IFS
audit_rules_networkconfig_modification.sh:                                unset $IFS
audit_rules_networkconfig_modification.sh:        unset $IFS
audit_rules_networkconfig_modification.sh:        unset $IFS
audit_rules_networkconfig_modification.sh:        unset $IFS
audit_rules_networkconfig_modification.sh:        unset $IFS
audit_rules_session_events.sh:        unset $IFS
audit_rules_session_events.sh:        unset $IFS
audit_rules_session_events.sh:        unset $IFS
audit_rules_sysadmin_actions.sh:        unset $IFS
audit_rules_time_adjtimex.sh:        unset $IFS
audit_rules_time_adjtimex.sh:        unset $IFS
audit_rules_time_adjtimex.sh:                                unset $IFS
audit_rules_time_clock_settime.sh:        unset $IFS
audit_rules_time_clock_settime.sh:        unset $IFS
audit_rules_time_clock_settime.sh:                                unset $IFS
audit_rules_time_settimeofday.sh:        unset $IFS
audit_rules_time_settimeofday.sh:        unset $IFS
audit_rules_time_settimeofday.sh:                                unset $IFS
audit_rules_time_stime.sh:        unset $IFS
audit_rules_time_stime.sh:        unset $IFS
audit_rules_time_stime.sh:                                unset $IFS
audit_rules_time_watch_localtime.sh:        unset $IFS
audit_rules_unsuccessful_file_modification.sh:        unset $IFS
audit_rules_unsuccessful_file_modification.sh:        unset $IFS
audit_rules_unsuccessful_file_modification.sh:                                unset $IFS
audit_rules_unsuccessful_file_modification.sh:        unset $IFS
audit_rules_unsuccessful_file_modification.sh:        unset $IFS
audit_rules_unsuccessful_file_modification.sh:                                unset $IFS
audit_rules_usergroup_modification.sh:        unset $IFS
audit_rules_usergroup_modification.sh:        unset $IFS
audit_rules_usergroup_modification.sh:        unset $IFS
audit_rules_usergroup_modification.sh:        unset $IFS
audit_rules_usergroup_modification.sh:        unset $IFS
```

Looking at the newest code base, it appears that unset IFS is the standard:
```
# grep -r 'unset IFS' *
shared/bash_remediation_functions/fix_audit_watch_rule.sh:	unset IFS
shared/bash_remediation_functions/fix_audit_syscall_rule.sh:	unset IFS
shared/bash_remediation_functions/fix_audit_syscall_rule.sh:	unset IFS
shared/bash_remediation_functions/fix_audit_syscall_rule.sh:				unset IFS
# grep -r 'unset $IFS' *
# 

```